components/browser/engine-gecko/src/test/java/mozilla/components/browser/engine/gecko/webnotifications/MockWebNotification.kt

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.engine.gecko.webnotifications
+
+import mozilla.components.support.test.mock
+import mozilla.components.test.ReflectionUtils
+import org.mozilla.geckoview.WebNotification
+
+fun mockWebNotification(
+    title: String,
+    tag: String,
+    text: String,
+    imageUrl: String,
+    textDirection: String,
+    lang: String,
+    requireInteraction: Boolean,
+    source: String? = null,
+    silent: Boolean = false,
+    vibrate: IntArray = IntArray(0)
```

Comment:
FWIW most of these are nullable, not sure if we care: https://searchfox.org/mozilla-central/source/mobile/android/geckoview/api.txt#2050-2059

Reply:
OK, updated to align with GV API.